docs/website/docs/community/index.md

```diff
@@ -14,15 +14,15 @@ Projects built by community members:
   [iree-turbine](https://github.com/iree-org/iree-turbine) and
   [sharktank](https://github.com/nod-ai/sharktank).
 
-* The [IREE Bare-Metal Arm Sample](https://github.com/iml130/iree-bare-metal-arm)
+* The [IREE Bare-Metal Arm Sample](https://github.com/iree-org/iree-bare-metal-arm)
   shows how to build IREE with the
   [Arm GNU Toolchain](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain)
   for bare-metal Arm targets using the open-source firmware libraries
   [CMSIS](https://github.com/ARM-software/CMSIS_5) and
   [libopencm3](https://github.com/libopencm3/libopencm3).
```

Comment:
We can move this down into the "Official repositories" section... or just merge the lists.